tests/bor/mining_test.go

```diff
@@ -55,6 +55,16 @@ var (
 // Example : CGO_CFLAGS="-D__BLST_PORTABLE__" go test -run ^TestMiningBenchmark$ github.com/ledgerwatch/erigon/tests/bor -v -count=1
 // In TestMiningBenchmark, we will test the mining performance. We will initialize a single node devnet and fire 5000 txs. We will measure the time it takes to include all the txs. This can be made more advcanced by increasing blockLimit and txsInTxpool.
 func TestMiningBenchmark(t *testing.T) {
+	// This test is disabled because it is not stable somehow.
```

Comment:
I don't love disabling tests, but given that it's in the integration path for bor, I concur this is relatively safe.